.travis.yml

```diff
@@ -0,0 +1,41 @@
+language: python
+
+python:
+  - 2.7
```

Comment:
Unfortunately, the tests do not work with 2.6 (because unittest lacks some features in 2.6 it has in 2.7). Nothing impossible to do, but that would require work. And I hope I'll be able to drop 2.6 support one day ;-)